backend/handlers/users/publicuser.go

```diff
@@ -10,21 +10,6 @@ import (
 	"gorm.io/gorm"
 )
 
-// PublicUserType is a struct for user data that is safe to send to the client for Profiles
-type PublicUserType struct {
-	Username              string `json:"username"`
-	DisplayName           string `json:"displayname" gorm:"unique;not null"`
-	UserType              string `json:"usertype"`
-	InitialAccountBalance int64  `json:"initialAccountBalance"`
-	AccountBalance        int64  `json:"accountBalance"`
-	PersonalEmoji         string `json:"personalEmoji,omitempty"`
-	Description           string `json:"description,omitempty"`
-	PersonalLink1         string `json:"personalink1,omitempty"`
-	PersonalLink2         string `json:"personalink2,omitempty"`
-	PersonalLink3         string `json:"personalink3,omitempty"`
-	PersonalLink4         string `json:"personalink4,omitempty"`
-}
-
```

Comment:
All of these fields are just being re-declared from the model. By making them a component type of the model User, we can access them all together.